directory.build.props

```diff
@@ -0,0 +1,27 @@
+<Project>
+  <PropertyGroup>
+    <Authors>.NET Foundation and Contributors</Authors>
+    <Copyright>Copyright (c) .NET Foundation and Contributors</Copyright>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>https://reactiveui.net</PackageProjectUrl>
+     <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)analyzers.ruleset</CodeAnalysisRuleSet>
+    <Owners>xpaulbettsx;ghuntley</Owners>
+    <Product>ReactiveUI ($(TargetFramework))</Product>
+    <PackageTags>mvvm;reactiveui;rx;reactive extensions;observable;LINQ;events;frp;xamarin;android;ios;mac;forms;monodroid;monotouch;xamarin.android;xamarin.ios;xamarin.forms;xamarin.mac;xamarin.tvos;wpf;net;netstandard;net461;uwp;tizen</PackageTags>
```

Comment:
Maybe remove the PackageTags, since you have those in the /src/Directory.Build.Props